test/1_stdlib/SetAlgebra.swift

```diff
@@ -0,0 +1,190 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+import StdlibCollectionUnittest
```

Comment:
To use `StdlibCollectionUnittest`, please move the test into `validation-test/stdlib/`.